objcgen/objcgenerator.cs

```diff
@@ -371,6 +376,59 @@ protected override void Generate (PropertyInfo pi)
 			ImplementMethod (setter, "set" + pi.Name, pi);
 		}
 
+		protected void Generate (FieldInfo fi)
+		{
+			bool read_only = ((fi.Attributes & FieldAttributes.InitOnly) == FieldAttributes.InitOnly);
+
+			headers.Write ("@property (nonatomic");
+			if (fi.IsStatic)
+				headers.Write (", class");
+			if (read_only)
+				headers.Write (", readonly");
+			else
+				headers.Write (", readwrite");
```

Comment:
This is the default `readwrite`, it does not harm to have it but it is a little uncommon to see in ObjC do we really want to have it?